src/components/Modal/Modal.tsx

```diff
@@ -42,10 +42,6 @@ const Modal: React.FC<ModalProps> = ({ children, onDismiss, isVisible, center })
 		}
 	}, [isVisible]);
 
-	if (!isVisible) {
-		return null;
-	}
-
```

Comment:
Had to remove that because the models were blinking. Need to do a proper fix for Android

Reply:
Got it.